build_cmake/configs/x86/darwin/config.cmake

```diff
@@ -0,0 +1,13 @@
+#
+# Public Domain 2014-present MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#  All rights reserved.
+#
+#  See the file LICENSE for redistribution information
+#
+
+cmake_minimum_required(VERSION 3.11.0)
```

Comment:
I'm wondering if there's a way in cmake to put this minimum version setting into a common config file that all other files could include - this line appears in multiple cmake config files.

Reply:
Thanks for raising that. I usually add `cmake_minimum_required` for each CMake file I write (out of habit).  I just revisited the docs and realized I really only need to define it once. I can remove the duplicate usages of it (other than the top CMakeLists).

Reply:
Fixed :)